tests/integration.rs

```diff
@@ -224,3 +225,32 @@ fn readlink_long(harness: Harness) {
 
 	assert_eq!(link, Path::new(&expected));
 }
+
+#[rstest]
+fn statfs(harness: Harness) {
+	let d = &harness.d;
+	let sfs = nix::sys::statfs::statfs(d.path()).unwrap();
+
+	assert_eq!(sfs.blocks(), 15751);
+	assert_eq!(sfs.blocks_free(), 15479);
+	assert_eq!(sfs.blocks_available(), 15479);
+	assert_eq!(sfs.files(), 8704);
+	assert_eq!(sfs.files_free(), 8692);
+	assert_eq!(sfs.maximum_name_length(), 255);
+
+	#[cfg(target_os = "freebsd")]
+	assert_eq!(sfs.block_size(), 4096);
+}
+
+#[rstest]
+fn statvfs(harness: Harness) {
+	let d = &harness.d;
+	let svfs = nix::sys::statvfs::statvfs(d.path()).unwrap();
+
+	//assert_eq!(svfs.block_size(), 32768);
```

Comment:
Why is this line commented out?  This is equivalent to statfs's f_iosize, so it probably shouldn't be included here at all.